google/auth/__init__.py

```diff
@@ -17,9 +17,10 @@
 import logging
 
 from google.auth._default import default, load_credentials_from_file
+from google.auth._default_async import default_async
```

Comment:
are we reusing load_credentials_from_file? Could that be added to async?

Reply:
Resolved

Reply:
Implemented inheritance from the sync classes with the default_async and load credentials from file to async so we do reuse the load_credentials_from_file method in both.